### DISCUSSION TITLE
How to learn Python?

### DISCUSSION THREAD

**Discussion by Foxglovve** [irgroup-classrooms/dis08-2021#4] [Q&A]
Hallo,

habt ihr Tipps und Empfehlungen, um Python zu lernen? Freie Tutorials oder sowas in der Art?

**Comment by ghost** [comment]
Die Seite ist ganz gut https://edabit.com/challenges

**Reply by Foxglovve** [reply]
Dankeschön!

**Reply by ghost** [reply]
Hello, 

ich hab mir autodidaktisch Python über w3schools und Codecademy beigebracht. 
Ist für Studenten zum Teil umsonst.

https://www.w3schools.com
https://www.codecademy.com/catalog
Have fun 🙋🏽‍♂️
